src/SpanInterface.php

```diff
@@ -44,9 +51,7 @@ public function activate(): SpanInterface;
     public function finish(): void;
 
     /**
-     * Returns the traceparent identifier (for distributed tracing).
-     * If the implementor does not want to support distributed tracing, this method
-     * MUST always return null.
+     * Returns distributed tracing headers, to allow trace correlation across service boundaries.
      */
-    public function toTraceparent(): ?string;
+    public function toTraceContextHeaders(): array;
```

Comment:
I lifted some of the wording from: https://opentelemetry.io/docs/concepts/signals/traces/#trace-context
OTEL _may_ provide more than one header value (traceparent + tracestate), and some implementations do not use the w3c headers.

Reply:
I understand why this change has been made. However, I do like the ability for the interface to grab an identifier that can be added to logs?

Maybe a `getTraceId` method?
-> Discord